install/kubernetes/cilium/values.yaml

```diff
@@ -2063,6 +2063,8 @@ preflight:
     effect: NoSchedule
   - key: node-role.kubernetes.io/master
```

Comment:
can you add a comment on this old key (e.g. replaced by new node-role.kubernetes.io/control-plance in k8s 1.26+) ? Thanks

Reply:
The comment has been updated, thanks for your review